python/graphstorm/model/lm_model/utils.py

```diff
@@ -15,11 +15,12 @@
 
     Utility functions for language support
 """
+from transformers import CONFIG_MAPPING
 from .hf_bert import wrap_hf_bert
 from .hf_bert import load_hfbert_model
 
 BUILTIN_HF_BERT = "bert"
-BUILTIN_LM_MODELS = [BUILTIN_HF_BERT]
+BUILTIN_LM_MODELS = CONFIG_MAPPING.keys()
```

Comment:
As you mentioned @wangz10 that deberta, GPT2 are not supported as the LM’s inputs and tokenizers are different, we need to exclude those models from this BUILTIN_LM_MODELS